Projects/UOContent/Items/Weapons/BaseMeleeWeapon.cs

```diff
@@ -2,16 +2,13 @@
 
 namespace Server.Items
 {
-    public abstract class BaseMeleeWeapon : BaseWeapon
+    [Serializable(0, false)]
```

Comment:
I might need to come up with some kind of codegen for this scenario. 
This class actually doesn't serialize a version variable. It passes through entirely.

Reply:
I've reverted this for now.